kclvm/sema/src/builtin/system_module.rs

```diff
@@ -1155,7 +1155,9 @@ register_json_member! {
 // ------------------------------
 
 pub const CRYPTO: &str = "crypto";
-pub const CRYPTO_FUNCTION_NAMES: &[&str] = &["md5", "sha1", "sha224", "sha256", "sha384", "sha512"];
+pub const CRYPTO_FUNCTION_NAMES: &[&str] = &[
+    "md5", "sha1", "sha224", "sha256", "sha384", "sha512", "uuid",
+];
```

Comment:
Lost `uuid` call for `register_crypto_member!`

Reply:
fixed